internal/login/login.go

```diff
@@ -0,0 +1,106 @@
+package login
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/launchdarkly/ldcli/internal/errors"
+)
+
+const ClientID = "e6506150369268abae3ed46152687201"
```

Comment:
do we want to commit this clientID?

Reply:
I believe it should be safe. The risk would be if we need to rotate it, which I don't think we would since it's more for identity than security -- another client (like vscode) would have its own client ID. If someone were to make a request with this value, they wouldn't get any valuable data in the response.